nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestBridgeCommentManager.java

```diff
@@ -91,7 +95,8 @@ protected DocumentModel getCommentedDocument(){
      * {@inheritDoc}
      * <p>
      * In the case of comment bridge we will ignore the check when we have an event of deletion comment, because the
-     * bridge checks the existence of the comment document and in this case we will end with an exception.
+     * bridge checks get the document to be able to root it into the good implementation
```

Comment:
"to route it to the good..." maybe?

Reply:
as discuss i changed `roote` by `dispatch` ;)